.pre-commit-config.yaml

```diff
@@ -0,0 +1,90 @@
+# To use:
```

Comment:
@tonynajjar can you review just this file? I did the rest but I could use your eye on this to make sure you're also happy with it